Add PowerPoint partial documents support documentation (#1619)
MicrosoftDocs/VBA-Docs: api/PowerPoint.Presentation.IsFullyDownloaded.md

---
title: Presentation.IsFullyDownloaded property (PowerPoint)
keywords: vbapp10.chm583138
f1_keywords:
- vbapp10.chm583138
ms.prod: powerpoint
api_name:
- PowerPoint.Presentation.IsFullyDownloaded
ms.date: 08/02/2022
ms.author: ononder
ms.localizationpriority: medium
---

# Presentation.IsFullyDownloaded property (PowerPoint)

**True** if the presentation has finished downloading all of the content. Read-only **Boolean.**

## Syntax

_expression_.**IsFullyDownloaded**

_expression_ A variable that represents a [Presentation](PowerPoint.Presentation.md) object.

## Remarks

> When you open a presentation with content that is large in size, PowerPoint may serve the document in parts as partial documents. This allows for documents to be opened, edited, and collaborated upon quickly, while the larger media parts (e.g., videos), continue to load in the background. Similarly, since media is handled separately from the rest of the document, collaboration is smoother when media is inserted during a collaboration session.

Reviewer: ```suggestion
When you open a presentation with content that is large in size, PowerPoint may serve the document in parts as partial documents. This allows you to open, edit, and collaborate on documents quickly, while the larger media parts (e.g., videos), continue to load in the background. Similarly, since media is handled separately from the rest of the document, collaboration is smoother when media is inserted during a collaboration session.
```
